### PR TITLE
feat : homepage

### PR DESCRIPTION
**Pull request by sudipkundu999**

### CODE REVIEW

File: homepage.css [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+}
+
+.hero-section{
+    height: 40rem;
+    width: 90%;
+    background-color: var(--black);
+    border-radius: 10px;
+    overflow: hidden;
+    display: flex;
+    box-shadow: rgba(100, 100, 111, 0.2) 0px 7px 29px 0px;
```
Comment (DivyanshSareen) [inline homepage.css:33]: can use var here

**Review by DivyanshSareen — APPROVED** [review]
Looks good. Good to merge

**Review by sanu29 — APPROVED** [review]
Looks good to me

**Review by zoyatanweer — APPROVED** [review]
neat code, good to merge

File: homepage.css (hunk 1 above)
Comment (sudipkundu999) [reply homepage.css:33]: It's intentional here as I've used this shadow/color only once

**Pull request merged**